src/test/java/de/retest/recheck/RecheckImplIT.java

```diff
@@ -29,36 +31,60 @@
 
 	@Test
 	void diff_should_be_created_accordingly() {
-		execute( "no-filter", Filter.FILTER_NOTHING );
+		execute( "no-filter", withIgnore( Filter.FILTER_NOTHING ) );
 	}
 
 	@Test
 	void diff_should_be_created_if_filtered() {
-		execute( "filter", Filters.parse( "matcher: retestid=same-id" ) );
+		execute( "filter", withIgnore( Filters.parse( "matcher: retestid=same-id" ) ) );
 	}
 
-	void execute( final String name, final Filter filter ) {
+	@Test
+	void diff_should_handle_legacy_spaces_accordingly() {
+		final MavenConformFileNamerStrategy fileNamerStrategy = spy( new MavenConformFileNamerStrategy() );
+		doReturn( RecheckImplIT.class.getName() + " legacy spaces" ).when( fileNamerStrategy ).getTestClassName();
+
+		execute( "with legacy spaces", RecheckOptions.builder() //
+				.fileNamerStrategy( fileNamerStrategy ) //
+				.build() );
+	}
+
+	@Test
+	void diff_should_handle_spaces_accordingly() {
+		execute( "with spaces", RecheckOptions.builder() //
+				.suiteName( RecheckImplIT.class.getName() + " spaces" ) //
+				.build() );
+	}
+
+	private RecheckOptions withIgnore( final Filter filterNothing ) {
```

Comment:
```suggestion
	private RecheckOptions withIgnore( final Filter ignore ) {
```

Or something like this because it isn't invoked exclusively with `Filter.FILTER_NOTHING` (see line 39).